packages/pn-personafisica-webapp/public/locales/it/notifiche.json

```diff
@@ -65,7 +65,10 @@
     "domicile_PEC": "Hai un indirizzo PEC? Aggiungilo e non riceverai più le raccomandate cartacee.",
     "domicile_EMAIL": "Aggiungi il tuo indirizzo e-mail: ti invieremo un messaggio ogni volta che ricevi una notifica.",
     "domicile_APPIO": "Hai l’app IO? Attiva il servizio \"Notifiche digitali\" per visualizzare e pagare le notifiche direttamente in app.",
+    "cancelled-alert-text": "Questa notifica è stata annullata dall’ente mittente. Puoi ignorarne il contenuto.",
+    "disclaimer-link": "Scopri di più",
     "payment": {
+      "cancelled-alert-text-payment": "Per l’eventuale rimborso di pagamenti legati a questa notifica, contatta l’ente mittente. ",
```

Comment:
there is an extra space at the end of the phrase

Reply:
it is wanted by ui

Reply:
remove it. and in te html where you use it, put a &nbsp; between this and the next word

Reply:
do it